Gulpfile.js

```diff
@@ -17,7 +19,16 @@ var publish      = require('publish-please');
 var del          = require('del');
 var fs           = require('fs');
 var path         = require('path');
+var opn          = require('opn');
+var connect      = require('connect');
+var readline     = require('readline');
+var spawn        = require('child_process').spawn;
+var serveStatic  = require('serve-static');
 var Promise      = require('pinkie');
+var markdownlint = require('markdownlint');
+// TODO: Uncomment when preparing the deploy-to-gh-pages task
+// var ghPages      = require('gulp-gh-pages');
```

Comment:
I was afraid of losing working code... 
But you're right, I'll keep it somewhere else so far